crane-cloud/frontend
# Github Authentication

# Description
This feature is about using github auth to be able to login or sign up to crane cloud.
## Type of change

Please delete options that are not relevant.


## Trello Ticket ID
https://trello.com/c/TCOm5uys

## How Can This Been Tested?
 checkout the branch and try to use github login instead.
you will need a client ID in you environment variable
Weather you registered  or not, you should be able to get in just by clicking login with github on the login page.

## Checklist:


# Screenshots

![git button](https://user-images.githubusercontent.com/69305400/132004341-7ab5b504-fbf8-490d-92a9-bd113c2d3e11.png)
